test/jdk/java/net/SocketPermission/SocketPermissionIm.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * @test
+ * @bug 8243376
+ * @summary SocketPermission implies(Permission p) spec allows, If the object was initialized with a single IP address and one of p's IP addresses is equal to this object's IP addr
+ * @run java -Dsun.net.inetaddr.ttl=0 SocketPermissionIm
+ */
```

Comment:
Instead of:
@run java
you need:
@run main/othervm

Also see further comment below.

Reply:
Done